riak_test/tests/stats_test.erl

```diff
@@ -66,28 +72,38 @@ query_stats(UserConfig, Port) ->
 
 confirm_initial_stats(StatData) ->
     %% Check for values for all meters to be 0 when system is initially started
-    [?assertEqual([0,0.0,0.0,0.0,0.0,0.0],
-                  proplists:get_value(StatType, StatData))
-                  || StatType <- [<<"block_get">>,
-                                  <<"block_put">>,
-                                  <<"block_delete">>,
-                                  <<"service_get_buckets">>,
-                                  <<"bucket_list_keys">>,
-                                  <<"bucket_create">>,
-                                  <<"bucket_delete">>,
-                                  <<"bucket_get_acl">>,
-                                  <<"bucket_put_acl">>,
-                                  <<"object_get">>,
-                                  <<"object_put">>,
-                                  <<"object_head">>,
-                                  <<"object_delete">>,
-                                  <<"object_get_acl">>,
-                                  <<"object_put_acl">>]],
+    ?assertEqual(106, length(StatData)),
+    [?assert(proplists:is_defined(StatType, StatData))
+     || StatType <- [<<"block_gets">>,
+                     <<"block_puts">>,
+                     <<"block_deletes">>,
+                     <<"service_get_buckets">>,
+                     <<"bucket_list_keys">>,
+                     <<"bucket_creates">>,
+                     <<"bucket_deletes">>,
+                     <<"bucket_get_acl">>,
```

Comment:
Nothing specific, but `get_acls` looks a bit odd to me. Let's discuss all the names next PR round.